.github/workflows/checks.yml

```
@@ -7,25 +7,32 @@ on:
       - '**'
   push:
     branches:
-      - 'main'
+      - 'v0'
 
 jobs:
   build_all:
     name: Build All
-    runs-on: ubuntu-22.04
+    runs-on:
+      - nscloud-ubuntu-20.04-amd64-2x32-with-cache
```

We probably don't need 32 gigs of RAM here, and maybe could use more cores. It would be nice to test a few options and see what gets decent performance without wasting money.

Random guess: 

```suggestion
      - nscloud-ubuntu-20.04-amd64-8x8-with-cache
```